src/main/resources/handlebars/JavaMicronaut/api.mustache

```diff
@@ -10,11 +10,17 @@ package {{package}};
 import com.fasterxml.jackson.databind.ObjectMapper;
 import io.micronaut.http.*;
 import io.micronaut.http.annotation.*;
+{{#useRxJava2}}
 import io.reactivex.Single;
+{{/useRxJava2}}
+{{#useRxJava3}}
+import io.reactivex.rxjava3.core.Single;
+{{/useRxJava3}}
 import io.swagger.v3.oas.annotations.*;
 import io.swagger.v3.oas.annotations.responses.*;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
```

Comment:
Looks like those was copy & pasted from another template. I do not see those to be used in generated code.